vault/namespaces_store_test.go

```diff
@@ -14,18 +14,18 @@ func TestNamespaceStore(t *testing.T) {
 	c, keys, root := TestCoreUnsealed(t)
 	s := c.namespaceStore
 
-	ctx := context.TODO()
+	ctx := namespace.RootContext(context.TODO())
```

Comment:
Here as well, as above.

Reply:
But maybe it is fine, since changing `FromContext(...)` will cause other problems... Hmm.